crates/autopilot/src/solvable_orders.rs

```diff
@@ -226,6 +238,63 @@ impl SolvableOrdersCache {
     pub fn last_update_time(&self) -> Instant {
         self.cache.lock().unwrap().orders.update_time
     }
+
+    /// Quotes all limit orders and sets the fee_amount for each one to the fee returned by the
+    /// quoting process. If quoting fails, the corresponding order is filtered out.
+    async fn set_limit_order_fees(&self, orders: Vec<Order>) -> Vec<Order> {
```

Comment:
I think even when caching gets introduced in a follow up PR time boxing the fee calculation might be needed.

Reply:
Good point. In the same module we also have a time limit on native prices. Should do something similar for getting as many quotes as possible and run both tasks at the same time.

Reply:
Thanks for letting me know @vkgnosis, that's good to know.